lib/buffer.js

```diff
@@ -61,8 +61,8 @@ export default class Buffer {
       this.markers[message.key] = marker
       this.messages.add(message)
       this.applyMarker(message, this.editors)
-      marker.onDidChange(({ newHeadBufferPosition, newTailBufferPosition, isValid }) => {
-        if (isValid) {
+      marker.onDidChange(({ oldHeadBufferPosition, newHeadBufferPosition, newTailBufferPosition, isValid }) => {
```

Comment:
Nvm, as you just pointed out this is the parameters from [here](https://atom.io/docs/api/v1.6.0/TextEditorMarker#instance-onDidChange).